### ISSUE TITLE
Foundation tax strategy

### ISSUE THREAD

**Issue by joshuavial** [enspiral/improvements#319]
Clarify a strategy for the way foundation manages taxes

**Comment by whiskeyfix** [comment]
@joshuavial @agentlewis heads up the next EF board meeting is next Wednesday 6th. We will be likely be spending some time on the agenda on Foundation financials.  Do you think we'll have an update / status we can take to that group by then? 
Fran and I are also working through the next venture agreement (including the financial contributions via cobudget) so this tax stuff we will also affect that.

**Comment by joshuavial** [comment]
@whiskeyfix - work to date is at https://github.com/enspiral/handbook/pull/108 - I'll checkin with @agentlewis and see if we can get this resolved

**Comment by whiskeyfix** [comment]
@joshuavial @agentlewis I can't  see anything in the link about tax, unless I'm missing something here?

We will need clarity on the tax relationships for both ventures and contributors for the upcoming value exchange updates. For both NZ and international entities and humans. I imagine we are going to need to get our approach ticked off by Deliotte / some expert on this. 

Can we set a deadline for a 1.0 to test? 

I have also added this ticket into the #balance-the-books label.
